config.yml

```diff
@@ -210,6 +210,7 @@ elrond:
 
 filecoin:
   api: https://api.filscan.io:8700/rpc/v1
+  explorer: https://filfox.info/api
```

Comment:
Can we drop /api here? And use that in the client instead

Reply:
removed